Sources/containertool/containertool.swift

```diff
@@ -28,112 +28,141 @@ enum AllowHTTP: String, ExpressibleByArgument, CaseIterable { case source, desti
         abstract: "Build and publish a container image"
     )
 
-    @Option(help: "Default registry for references which do not specify a registry")
-    private var defaultRegistry: String?
-
-    @Option(help: "Repository path")
-    private var repository: String?
-
     @Argument(help: "Executable to package")
     private var executable: String
 
-    @Option(help: "Resource bundle directory")
-    private var resources: [String] = []
+    /// Options controlling the locations of the source and destination images
+    struct RepositoryOptions: ParsableArguments {
+        @Option(help: "Default registry for image references which do not specify one")
+        var defaultRegistry: String?
 
-    @Option(
-        help: ArgumentHelp(
-            "[DEPRECATED] Default username, used if there are no matching entries in .netrc. Use --default-username instead.",
-            visibility: .private
-        )
-    )
-    private var username: String?
+        @Option(help: "Destination image reference")
+        var repository: String?
 
-    @Option(help: "Default username, used if there are no matching entries in .netrc")
-    private var defaultUsername: String?
+        @Option(help: "Destination image tag")
+        var tag: String?
 
-    @Option(
-        help: ArgumentHelp(
-            "[DEPRECATED] Default password, used if there are no matching entries in .netrc.   Use --default-password instead.",
-            visibility: .private
-        )
-    )
-    private var password: String?
+        @Option(help: "Base image reference")
+        var from: String?
+    }
 
-    @Option(help: "Default password, used if there are no matching entries in .netrc")
-    private var defaultPassword: String?
+    @OptionGroup(title: "Source and destination repository options")
+    var repositoryOptions: RepositoryOptions
 
-    @Flag(name: .shortAndLong, help: "Verbose output")
-    private var verbose: Bool = false
+    /// Options controlling how the destination image is built
+    struct ImageBuildOptions: ParsableArguments {
+        @Option(help: "Directory of resources to include in the image")
```

Comment:
```suggestion
        @Option(help: "The list of directories with resources to include in the image")
```
should this be plural - the resources is an array?

Reply:
`--resources` takes a single directory or file as an argument, but you can specify it multiple times e.g. `--resources foo --resources bar`.   

The plugin uses this option automatically if the product includes a resource bundle in `Package.swift`.   You can only have one resource bundle per target, but the user could specify that additional resources should be added to the bundle with this flag.